azure/pipeline/azure_tools.py

```diff
@@ -0,0 +1,35 @@
+import planetary_computer
+
+def get_fs(account='nrel', container='oedi'):
+    return planetary_computer.get_adlfs_filesystem(account, container)
+
+def get_size(path, units='B'):
+    fs = get_fs()
+    size = fs.du(path, total=True)
+    match units:
```

Comment:
This syntax requires Python version >=3.10, 

We need to upgrade our CI testing Python version, it's 3.9, which raised syntax error
```bash
./azure/pipeline/azure_tools.py:9:11: E999 SyntaxError: invalid syntax
    match units:
          ^
1     E999 SyntaxError: invalid syntax
1
Error: Process completed with exit code 1.
```

Reply:
changed to if structure